aspnetcore/fundamentals/configuration.md

```diff
@@ -352,3 +353,4 @@ results in an exception. An exception will be thrown if you specify a command-li
 * [Working with Multiple Environments](environments.md)
 * [Safe storage of app secrets during development](../security/app-secrets.md)
 * [Dependency Injection](dependency-injection.md)
+* [Azure Key Vault Configuration Provider](xref:security/key-vault-configuration)
```

Comment:
Sure thing. You said in the main issue to link from the Config doc, so I thought you meant "**only** link from the config doc." My bad.